glean-core/android/src/test/java/mozilla/telemetry/glean/GleanTest.kt

```diff
@@ -252,28 +252,6 @@ class GleanTest {
         assertTrue(gleanDir.delete())
     }
 
-    @Ignore("This should probably be implemented in the Rust side.")
-    @Test
-    fun `Don't send metrics if not initialized`() {
```

Comment:
Can you check if this was implemented on the Rust side? If not, can you file a follow-up bug for creating this test?

Reply:
Bug filed [here](https://bugzilla.mozilla.org/show_bug.cgi?id=1569609)